### PR TITLE
ENH: Mark received image data scalars as modified

### PR DESCRIPTION
**Pull request by Sunderlandkyl**
Triggers updates for attributes such as scalar range.

### CODE REVIEW

File: Converter/igtlioImageConverter.cxx [hunk 1]
```diff
@@ -286,7 +288,12 @@ int igtlioImageConverter::IGTLToVTKImageData(igtl::ImageMessage::Pointer imgMsg,
       }
 
     }
-  
+
+  // Mark scalars as modified
```
Comment (lassoan) [inline Converter/igtlioImageConverter.cxx:292]: Add to the comment that this is needed for example for getting up-to-date scalar range.

**Review by lassoan — COMMENTED** [review]
Looks good, thank you, just added a small comment.

**Comment by lassoan** [comment]
Since you are modifying this module anyway, can you check if you can add a Send method for one-off sending of a node (as requested by a user). I think it was a public method of the connector.

**Comment by Sunderlandkyl** [comment]
Sure, I'll take a look at it.

**Pull request merged**